### DISCUSSION TITLE
Google Home - Automation

### DISCUSSION THREAD

**Discussion by franklinkf** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#925] [General]
Today I installed this programme, but I think the automations/routines I setup in Google Home is not supported.

Can you please confirm?

**Comment by im-coder-lg** [comment]
Yeah, I'm afraid you are right about that. See, this project uses the very-limited Google Assistant SDK, which gives only half of the features on a phone. So things like Hotwire activation and the like are added manually. But some other issues such as personal results, automation and so on have to be enabled at Google's side, but unfortunately this can't happen. Previously, Personal Results could be enabled via a small loophole, and I had made a simple list of steps to achieve that on GitHub Gist, but since this year, it seems like Google patched it up. Google Home Automation Control faces a similar issue due to the limits of the SDK. So, we won't know when support for this could be added.

TL;DR: due to SDK limitations, things such as Personal Results(which used to work via loopholes) and Google Home Automations can't work, due to Google limiting it.

**Comment by franklinkf** [comment]
Thank you for prompt and detailed reply

**Reply by im-coder-lg** [reply]
Anytime! I think there has been a discovery of another similar loophole to the Personal Results feature. I will try it and update my gist, and perhaps even check if it helps with basic Google Home Automation later on, and will report back. This looks interesting, and so do the loopholes.
